src/runtime/c_runtime_api.cc

```diff
@@ -106,6 +106,18 @@ void DeviceAPI::FreeWorkspace(TVMContext ctx, void* ptr) {
   FreeDataSpace(ctx, ptr);
 }
 
+TVMStreamHandle DeviceAPI::CreateStream(TVMContext ctx) {
+  throw std::runtime_error("Device does not support stream api.");
+}
+
+void DeviceAPI::FreeStream(TVMContext ctx, TVMStreamHandle stream) {
+  throw std::runtime_error("Device does not support stream api.");
+}
+
+void DeviceAPI::SyncStreams(TVMContext ctx, TVMStreamHandle event_src, TVMStreamHandle event_dst) {
```

Comment:
Change to SyncStreamFromTo (so argument position is clear from the function name)